pvr.zattoo/pvr.zattoo.txt

```diff
@@ -0,0 +1 @@
+pvr.zattoo https://github.com/rbuehlma/pvr.zattoo 18.0.7
```

Comment:
For the K18 development phase, replace the version tag by "master", please.

Reply:
Done

Reply:
>  Done but please note that the current master is missing the libcurl removal

Then you should add it. Once this PR is merged, we pull whatever you have on master. It's your responsibility that there is something useful.  ;-)

Reply:
I will, as soon as https://github.com/xbmc/xbmc/pull/12728 got merged. So please wait with merging this PR.

Reply:
sure, absolutely no problem.